zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/archiver/OpenSearchRepositoryIT.java

```diff
@@ -81,6 +87,81 @@ void shouldDeleteDocuments() throws IOException {
         .isEqualTo("3");
   }
 
+  @Test
+  void shouldSetIndexLifeCycle() throws IOException {
+    // given
+    final var indexName = UUID.randomUUID().toString();
+    final var repository = createRepository();
+    testClient.indices().create(r -> r.index(indexName));
+
+    final OpensearchEngineClient engineClient = new OpensearchEngineClient(testClient);
+    final OpenSearchGenericClient genericClient =
+        new OpenSearchGenericClient(testClient._transport(), testClient._transportOptions());
+
+    retention.setEnabled(true);
+    retention.setPolicyName("operate_delete_archived_indices");
+
+    // when
+    createLifeCyclePolicy(engineClient);
```

Comment:
:wrench: Considering the `engineClient` is only used for this method call, why not just create it in there, and remove the argument? Since `testClient` is available anyway to the whole class.

Reply:
It is also used in indexContainsRetentionPolicy() but I will see if it is possible to refactor after rewriting the test 🤔